sklearn/feature_selection/tests/test_variance_threshold.py

```diff
@@ -12,15 +12,19 @@
         [1, 1, 2, 4, 0]]
 
 
-def test_zero_variance():
+@pytest.mark.parametrize(
+    "args",
+    [[[0, 1, 2, 3]],
+     [[0, 1], [0, 1]]])
+def test_zero_variance(args):
     # Test VarianceThreshold with default setting, zero variance.
 
     for X in [data, csr_matrix(data), csc_matrix(data), bsr_matrix(data)]:
         sel = VarianceThreshold().fit(X)
         assert_array_equal([0, 1, 3, 4], sel.get_support(indices=True))
```

Comment:
I don't think it's good to use pytest.mark.parametrize here, because we'll run same test for multiple times.